scene/main/scene_tree.cpp

```diff
@@ -1962,7 +1962,7 @@ bool SceneTree::is_using_font_oversampling() const {
 
 SceneTree::SceneTree() {
 
-	singleton = this;
+	if (singleton == NULL) singleton = this;
```

Comment:
So we can no longer instantiate new `SceneTree`s to take over the main loop? Owwww :'(

Reply:
No, previously, instantiating a `SceneTree` and freeing it immediately afterwards resulted in `SceneTree::get_singleton() == NULL`. This change (and the change a few lines down) is intended to keep only the first SceneTree as the singleton tree.